### PR TITLE
Proof of concept for multi viewer

### PR DESCRIPTION
**Pull request by helblazer811**
I have a split viewer that works. It uses the same mouse movements for all similar to linkedViewer. Some things don't work correctly like backgroundColor.

### CODE REVIEW

**Comment by helblazer811** [comment]
Do not merge this yet i should be able to add a few new features soon

**Comment by helblazer811** [comment]
This is actually still bugged so ignore this

File: 3Dmol/glviewer.js [hunk 1]
```diff
@@ -1202,6 +1206,32 @@ $3Dmol.GLViewer = (function() {
...
+            if(Array.isArray(axis)){
+               axis = {x:axis[0],y:axis[1],z:axis[2]} 
+            }
+
+            var viewer = this;
+
+            
+            spinInterval = setInterval(
+                function(){
+                    console.log("rotating")
```
Comment (dkoes) [inline 3Dmol/glviewer.js]: need to remove debug code

File: 3Dmol/glviewer.js [hunk 2]
```diff
@@ -1202,6 +1206,32 @@ $3Dmol.GLViewer = (function() {
             ret.normalize();
             return ret;
         };
+        var spinInterval;
+        this.spin = function(spin,axis,interval){
```
Comment (dkoes) [inline 3Dmol/glviewer.js]: I was thinking of overloading the first argument to serve the purpose of both spin and axis.
```
spin() 
spin(true)
spin('y')
```
would all do the same thing
`spin(false)` deactivates

Comment (helblazer811) [inline 3Dmol/glviewer.js]: I was also thinking that the axis could define speed as well [2,0,0] would spin twice the default on the x axis as [1,0,0]. With just supplying the interval you can only specify one speed for all of the axis and with a high interval there is visible skipping. This way it is continuous and you can specify different speeds for all 3 axis.

**Pull request merged**